packages/formation-react/src/components/ErrorableCheckbox/ErrorableCheckbox.jsx

```diff
@@ -88,13 +89,27 @@ ErrorableCheckbox.propTypes = {
    */
   name: PropTypes.string,
   /**
-   * Label for the checkbox
+   * Label for the checkbox.
+   * IF you have a SEPARATE HEADING (outside this component) that “labels” this
+   * checkbox, do NOT pass {<span class="usa-sr-only">...</span>} here to just
+   * visually hide this label.
+   * Instead, pass an empty string ("") here, and pass that external heading‘s
```

Comment:
Could we make this not a required prop? Forcing people to pass an empty string feels weird. We're also still rendering an empty label element, which maybe we don't want to do? Not sure how @1Copenut feels about that.

Or we could write a custom prop type here that says you need a label if you aren't using the aria labelled by prop.

Reply:
Custom-validators now added to label & ariaLabelledBy -- either one is required but not both.